dashboard/management/commands/getjwk.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+from datetime import datetime
+
+from django.core.management.base import BaseCommand, CommandParser
+from django.http import JsonResponse
+
+from dashboard import lti_new
+from dashboard.keyfileutils import KeyFileUtils
+
+
+class Command(BaseCommand):
+    help = 'Generate public key JWK as JSON using ' \
+           'the currently configured public key.'
+    baseNameOption = 'basename'
+    dumpOption = 'dump'
+
+    def add_arguments(self, parser: CommandParser):
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument(
+            f'--{self.baseNameOption}',
+            default=datetime.now().strftime(KeyFileUtils.timestampFormat),
+            type=str,
+            required=False,
+            help='Base file name of the key file(s) generated. '
+                 'Default: timestamp of '
+                 f'"{KeyFileUtils.timestampFormat.replace("%", "%%")}"',
+            dest=self.baseNameOption)
+        group.add_argument(
+            f'--{self.dumpOption}',
+            action='store_true',
+            default=False,
+            required=False,
+            help='Dump the JWK JSON to stdout instead of writing it to a file',
+            dest=self.dumpOption)
+
+    def handle(self, *args, **options: dict):
+        jwkResponse: JsonResponse = lti_new.get_jwks(None)
```

Comment:
Should we change `lti_new.get_jwks` so it doesn't require you to pass an unnecessary parameter?

Reply:
Yes, I could do that.  I considered it, but it wasn't strictly necessary in the scope of implementing this feature.  Since you're thinking along the same lines and the argument of `lti_new.get_jwks()` isn't even used, I think it's worth doing.

Reply:
Although I can use `*_` to let Django pass a Request object without error, yet not require an argument, this doesn't seem ideal.  This tool uses `lti_new.get_jwks()` to avoid repeating the call to `pylti1p3` methods.  However, since this method returns JSON in a HTTP response, the tool must decode that to use it.

It would probably be better to refactor `lti_new.get_jwks()` into two methods.  One for Django to call that returns a JSON HTTP response and one to get the payload for that response.

Reply:
I'm not really following, but if this is going to require a lot of tinkering with `lti_new`, I would vote to defer this, so you can get the people who worked on that module originally involved in whatever changes you're thinking about.

Reply:
I've updated `lti_new` in the past, so I think it's not a problem.

Reply:
I've updated `lti_new` in the past, so I think it's not a problem.